rwlib/CMakeLists.txt

```diff
@@ -41,10 +41,9 @@ SET(RWLIB_SOURCES
 	"source/job/WorkContext.cpp"
 	)
 
-add_library(rwlib
+set_source_files_properties(source/gl/gl_core_3_3.c PROPERTIES COMPILE_FLAGS -w)
```

Comment:
This sounds like a reasonable idea. Sorry for the inconvenience :disappointed: 
There is already #59 

Maybe glloadgen is simply considered "complete"? I'll try to find out or look for alternatives. I'll also try to convince them to include styles which might not be nice, but work error-free for modern C++. I'll also still look into the cmake alternative. (but everyone else is free to do it too, as I'm super busy during the next months)